### ISSUE TITLE
Full-width banner: Design system link is broken in Create > form description 

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#10063]
## Describe the defect

![Screen Shot 2022-08-04 at 10 33 10 AM](https://user-images.githubusercontent.com/85581471/182914368-57bcb962-02e6-4812-959c-a89c56e35ece.png)

On, https://prod.cms.va.gov/node/add/banner the VA Design System link (https://design.va.gov/components/alertboxes) throws a 403. 

I think the link should point to https://design.va.gov/components/banner

## AC / Expected behavior
- [ ] VA Design System link points to working link re: Full-width banners

## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS workstream (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [X] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by EWashb** [comment]
@jilladams can you refresh my memory on this one? i see I added the sprint 65 label but I didn't prioritize this one. Was there a slack thread about this? @dawnpruitt

**Comment by jilladams** [comment]
Yes - tl;dr: I just tripped over this during some research, but it's not high priority. Whenever you all are looking for work, feel free, I just cut a ticket for awareness. Not urgent.

**Comment by EWashb** [comment]
Thank you. You're the best and I appreciate your help pulling that memory from the backlog in my brain.
